docs/aws_s3_compat.md

```diff
@@ -290,41 +309,6 @@ See also `GetObject` and other method parameters.
 | 🟡 | PutBucketPolicy         | See ACL limitations         |
 | 🔵 | PutBucketReplication    |                             |
 
-By default bucket ACLs is disabled. See details [here](https://docs.aws.amazon.com/AmazonS3/latest/userguide/about-object-ownership.html).
```

Comment:
This should be noted in the ACL section with reference to "ownership controls".

Reply:
Moved